dxr/plugins/xpidl/mappings.py

```diff
@@ -0,0 +1,29 @@
+from dxr.filters import LINE
+from dxr.indexers import STRING_PROPERTY
+
+# Like qualified line needle, but has no qualname
+LINE_NEEDLE = {
+    'type': 'object',
+    'properties': {
+        'name': STRING_PROPERTY,
+        'start': {
+            'type': 'integer',
+            'index': 'no'  # just for highlighting
+        },
+        'end': {
+            'type': 'integer',
+            'index': 'no'
+        }
+    }
+}
+
+mappings = {
+    LINE: {
+        'properties': {
+            'xpidl_var_decl': LINE_NEEDLE,
```

Comment:
Yes, not all .idl files are xpidl files -- for example, some are MSIDL (?), Microsoft's IDL-ish variant: https://dxr.allizom.org/mozilla-central/search?q=%22%3A+IUnknown%22+ext%3Aidl